scripts/gulpfiles/appengine_tasks.js

```diff
@@ -47,8 +47,8 @@ function copyStaticSrc(done) {
  */
 function copyAppengineSrc() {
   const appengineSrc = [
-      path.join(demoStaticTmpDir, 'appengine/**/*'),
-      path.join(demoStaticTmpDir, 'appengine/.gcloudignore'),
+      `${demoStaticTmpDir}/appengine/**/*`,
+      `${demoStaticTmpDir}/appengine/.gcloudignore`,
```

Comment:
Based on my reading https://gulpjs.com/docs/en/api/vinyl/#normalization-and-concatenation,
is that why the call to `path.join` here isn't necessary?

Reply:
More specifically because of [the documentation for using globs with `src`](https://gulpjs.com/docs/en/getting-started/explaining-globs/), though I suspect the two are connected.